internal/pkg/errors/errors.go

```diff
@@ -1,21 +1,27 @@
 package errors
 
 import (
-	"github.com/pkg/errors"
+	"errors"
+
+	perrors "github.com/pkg/errors"
 )
 
 func New(msg string) error {
-	return errors.New(msg)
+	return perrors.New(msg)
 }
 
 func Wrap(err error, msg string) error {
-	return errors.Wrap(err, msg)
+	return perrors.Wrap(err, msg)
 }
 
 func Wrapf(err error, fmt string, args ...any) error {
-	return errors.Wrapf(err, fmt, args...)
+	return perrors.Wrapf(err, fmt, args...)
 }
 
 func Errorf(fmt string, args ...any) error {
-	return errors.Errorf(fmt, args...)
+	return perrors.Errorf(fmt, args...)
+}
+
+func Join(errs ...error) error {
+	return errors.Join(errs...)
```

Comment:
So `errors.Join()` is not part of `github.com/pkg/errors`?

Reply:
Nope. It's part of the standard library like `fmt` or `os`.